dist/components/list/list.dust

```diff
@@ -0,0 +1,11 @@
+<ul class="c-list">
```

Comment:
I just noticed that the CSS includes modifier on the parent class for `c--ul` and `c--ol` but this dust template doesn't facilate a means to add those modifiers.

Maybe not a huge deal for now. Something to fix in Stencil 2.0?

Possibility an alternative approach (food for thought) would be to remove the `ul`/`ol` modifier classes and rely on the markup instead. Make the `c-list` tag itself be modifiable to `ul` or `ol` as necessary.